nab/optimizer.py

```diff
@@ -37,115 +39,55 @@ def optimizeThreshold(args):
         "score"     (float)   The score from the objective function given the
                               threshold.
   """
-  optimizedThreshold, optimizedScore = twiddle(
-    objFunction=objectiveFunction,
-    args=args,
-    initialGuess=0.5,
-    tolerance=.0000001)
-
-  print "Optimizer found a max score of {} with anomaly threshold {}.".format(
-    optimizedScore, optimizedThreshold)
+  (pool,
+   detectorName,
+   profileName,
+   costMatrix,
+   resultsDetectorDir,
+   resultsCorpus,
+   corpusLabel,
+   probationaryPercent,
+   scoreFlag) = args
```

Comment:
Does this change the expected arrangement of `args` into `optimizeThreshold()`? If so, the comments should be updated. For example, `tolerance` was an argument for the old twiddle algorithm.

Reply:
Argument comments and order updated.